jest.config.js

```diff
@@ -0,0 +1,9 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+
+  clearMocks: true,
+
+
+};
```

Comment:
Change this to `export` or `export default`  etc. like you did in the other test files